library/modules/Filesystem.cpp

```diff
@@ -154,17 +154,20 @@ bool Filesystem::stat (std::filesystem::path path, std::filesystem::file_status
 
 bool Filesystem::exists (std::filesystem::path path) noexcept
 {
-    return std::filesystem::exists(path);
+    std::error_code ec;
+    return std::filesystem::exists(path, ec);
```

Comment:
As far as I can tell, the return value of `exists` is unspecified if an error is detected. I think I'd prefer for this function to return `false` if an error is detected, instead of returning an unspecified value.